dependencies/vaticle/repositories.bzl

```diff
@@ -21,5 +21,5 @@ def vaticle_dependencies():
     git_repository(
```

Comment:
In the new version of dependencies, the auth token passed to the deploy_npm rule, used to deploy the protocol npm package, is now masked in the rule's output logs.